### ISSUE TITLE
Spezial characters

### ISSUE THREAD

**Issue by erikradisch** [infoscout/weighted-levenshtein#13]
Hey!
Awesome module! It is exactly what I need. I only have one problem. Do I understand it right, that it works only with ascii-code? I tried special characters and they lead into an error...

**Comment by RevolutionTech** [comment]
That's correct. `weighted-levenshtein` only support ASCII unfortunately. A PR to add UTF-8 support would be welcome though!

**Comment by B3AU** [comment]
Mapping with chr seems to work:

`mapping = {s:chr(i+1) for i,s in enumerate(symbols)}`

\x00 causes trouble, hence the +1

**Comment by LEFTazs** [comment]
> That's correct. `weighted-levenshtein` only support ASCII unfortunately. A PR to add UTF-8 support would be welcome though!

The problem might be the line endings. Linux uses \n, while Windows uses \r\n.
@RevolutionTech  Is there any way line endings might be part of the Damerau code logic?
